server/webapp/WEB-INF/vm/auth/login.vm

```diff
@@ -123,6 +123,11 @@
                 </div>
                 #end
                 <div class='clear-both'><!-- Clear floats --></div>
+
+#foreach ($authentication_plugin_id in $authentication_plugin_registry.getAuthenticationPlugins())
```

Comment:
I guess we should think about the UI of this, as well. Should the administrator be able to set the default way of login?